lib/models/page.test.ts

```diff
@@ -71,21 +71,25 @@ describe('Page', () => {
 
   describe('.isPublic', () => {
     describe('with a public page', () => {
-      test('should return true', done => {
-        Page.findOne({ path: '/grant/public' }, (err, page) => {
-          expect(err).toBeNull()
-          expect(page.isPublic()).toBe(true)
-          done()
+      test('should return true', () => {
```

Comment:
All changes of `test` files are because of

- fixed by linter of `jest/no-test-callback`
- and then fixed manually `promise/param-names`